src/sonic-yang-models/yang-models/sonic-tunnel.yang

```diff
@@ -0,0 +1,117 @@
+module sonic-tunnel {
+    yang-version 1.1;
+    namespace "http://github.com/Azure/sonic-tunnel";
```

Comment:
should we use sonic-net to replace Azure?

Reply:
let's do the change together with other models.